panic: runtime error: slice bounds out of range [:-1]

Here's the snippet:
```
➜  dh-make-golang make -allow_unknown_hoster code.cloudfoundry.org/bytefmt  
panic: runtime error: slice bounds out of range [:-1]

goroutine 1 [running]:
main.shortHostName(0x7fff2089d3aa, 0x1d, 0x967f01, 0x1, 0x0, 0x1, 0xc0004f0320)
	github.com/Debian/dh-make-golang/make.go:546 +0x7dd
main.debianNameFromGopkg(0x7fff2089d3aa, 0x1d, 0x1, 0xc0001aa401, 0x0, 0x0)
	github.com/Debian/dh-make-golang/make.go:565 +0x122
main.execMake(0xc0000860a0, 0x2, 0x2, 0x0)
	github.com/Debian/dh-make-golang/make.go:784 +0x6a2
main.main()
	github.com/Debian/dh-make-golang/main.go:60 +0x317
```

--- a/make.go
+++ b/make.go
@@ -543,7 +543,7 @@ func shortHostName(gopkg string, allowUnknownHoster bool) (host string, err erro
 	default:
 		if allowUnknownHoster {
 			suffix, _ := publicsuffix.PublicSuffix(host)
-			host = host[:len(host)-len(suffix)-len(".")]
+			host = fqdn[:len(fqdn)-len(suffix)-len(".")]
 			log.Printf("WARNING: Using %q as canonical hostname for %q. If that is not okay, please file a bug against %s.\n", host, fqdn, os.Args[0])
 		} else {
 			err = fmt.Errorf("unknown hoster %q", fqdn)

